aboutme/style.css

```diff
@@ -0,0 +1,56 @@
+html {
+  line-height: 1.15;
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+h1{
+  padding-top: 100px;
+  font-size:4em;
+  text-align: center;
+  margin: 0.67em 0;
+  font-family: 'Rubik Mono One', sans-serif;
+  margin: 10px;
+}
+
+h4{
+  font-family: 'Ubuntu', sans-serif;
+  text-align: center;
+}
+
+body {
+  margin: 0;
+}
+
+section{
+  font-family: 'Raleway', sans-serif;
+  padding: 20px;
+  float: left;
+  width: 50%;
+  color:gray;
+  display: inline-block;
+}
+
+aside {
+  float: right;
+  width: 30%;
+  padding: 10px;
+  border: 1px dotted red;
+  font-family: 'Space Mono', monospace;
+  margin-right: 30px;
+   }
+
+a {
+  background-color: transparent;
+  -webkit-text-decoration-skip: objects;
```

Comment:
This was part of  'CSS normalize' script that I found somewhere. I believe it made the background transparent to clear out any browser default colors.